### DISCUSSION TITLE
Cannot access underscore folder/file with nojekyll

### DISCUSSION THREAD

**Discussion by Tokenyet** [community/community#22975] [Other Feature Feedback, Questions, & Ideas]
<p>The site “looks” fine when using desktop version, but If opened the emulator in devTools, and select any phone to see More(…), and refresh the page, I’ll see the following error:</p>
<p>GET <a href="https://minecraftscepter.github.io/_next/static/runtime/webpack-4b444dab214c6491079c.js" target="_blank" rel="nofollow noopener">https://minecraftscepter.github.io/_next/static/runtime/webpack-4b444dab214c6491079c.js</a> net::ERR_ABORTED 404</p>
<p>This is my repository:</p>
<p><a href="https://github.com/MinecraftScepter/MinecraftScepter.github.io/" rel="nofollow noopener">https://github.com/MinecraftScepter/MinecraftScepter.github.io/</a> </p>
<p>The site was working properly when I tested in local, and I checked the file that existed in repository, but I cannot access with the <a href="https://minecraftscepter.github.io/" rel="nofollow noopener">https://minecraftscepter.github.io/</a>. Thanks for taking a look at this!</p>

**Comment by Tokenyet** [comment, marked as answer]
<p>Never mind, uploaded twice with no-change commit, and the site works! It might trigger something to rebuild.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/tokenyet">@tokenyet</a>, thanks for coming back and sharing your fix.  We’re looking forward to seeing you around!</p>
